### PR TITLE
Geonames API integration

### PR DESCRIPTION
**Pull request by sdreher**
Switch from Google Places API to the Geonames API.

### CODE REVIEW

**Comment by sdreher** [comment]
This PR requires an update to `local_settings.py`. Please review, but wait to merge until Monday.

File: media/js/xeditable/bootstrap-editable-place.js [hunk 1]
```diff
@@ -38,55 +44,54 @@ Place editable input.
...
-                        }
-                    });
-                }
-            });
+        markupName: function(obj) {
+            let str = '<div class="separator"><div>' + obj.name;
+            if (obj.adminName1) {
+                str += ', ' + obj.adminName1;
+            }
+            str += '</div><div>' + obj.countryName + '</div>';
```
Comment (nikolas) [inline media/js/xeditable/bootstrap-editable-place.js:52]: Is there a missing closing `</div>` here? I'm counting three open `<div>` tags, two closing ones.

Comment (sdreher) [reply media/js/xeditable/bootstrap-editable-place.js]: good catch! i've resolved.

**Comment by sdreher** [comment]
Okay - I've updated the application settings with the API key. This can be merged.

**Pull request merged**